tests/scripts/all.sh

```diff
@@ -822,6 +822,22 @@ component_test_platform_calloc_macro () {
     make test
 }
 
+component_test_malloc_0_null () {
+    msg "build: malloc(0) returns NULL (ASan+UBSan build)"
+    scripts/config.pl full
+    scripts/config.pl unset MBEDTLS_MEMORY_BUFFER_ALLOC_C
```

Comment:
Since https://github.com/ARMmbed/mbedtls/pull/2469 you don't have to do that.

Reply:
I know, but that's only in mbedtls. The change isn't in mbed-crypto yet and this PR is for mbed-crypto.

Reply:
Right, sorry!